### ISSUE TITLE
Feature request: replace existing table columns

### ISSUE THREAD

**Issue by KatieMFritz** [verbb/element-index-defaults#2]
I totally understand if you decide this isn't worth the risk of someone wiping out their custom configuration and getting mad. 😄 

However, it would be kind of nice to have this option:
1. Configure plugin settings
2. Button to "Replace existing table columns"
3. Confirmation modal: "Are you sure? This will overwrite your existing table configurations for [Assets]. Yes, replace existing columns/Cancel"
4. All table columns for that element type get updated to plugin settings
5. Uninstalling or disabling the plugin returns columns to either Craft defaults or previous configuration, whichever is easier/makes more sense to y'all

**Comment by engram-design** [comment]
Yep, I think this is a great FR and super-useful! We'll implement this at some stage.

**Comment by engram-design** [comment]
I know this is an old issue - I'm just waiting for Craft 4 which will (apparently) bring changes to the element indexes. Here's hoping we can handle this through defaults!
